storm/src/main/storm/mesos/NimbusScheduler.java

```diff
@@ -102,7 +102,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       case TASK_LOST:
       case TASK_ERROR:
         LOG.info("Received status update: {}", taskStatusToString(status));
```

Comment:
@erikdw Thanks for the explanation, I see a lot of references to `TASK_ERROR` and `TASK_RUNNING` in the Apache Mesos code.

@JessicaLHartog, @erikdw do you mean changing the existing LOG.info for TASK_ERROR to LOG.warn and introducing logging for TASK_FAILED, TASK_KILLED etc (if they don't have logging already).